### PR TITLE
Vertex AI Model Registry - AutoML model management notebook

### PR DESCRIPTION
**Pull request by inardini**
<br>
This PR will add a notebook to show the model versioning capabilities of Vertex AI Model Registry with AutoML model. The notebook uses this [public dataset](http://mlg.ucd.ie/datasets/bbc.html) of BBC news articles which has been already utilised in [this](https://www.cloudskillsboost.google/focuses/1749?parent=catalog) Google Cloud Self-packed lab. 
<br><br><br>

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb [hunk 1]
```diff
@@ -0,0 +1,2202 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>Versioning -&gt; versioning</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/995/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>Separate into 4 cells</p><p>Add links to BQ and Dataproc pricing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/995/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>combine 2 text cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/995/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>you will -&gt; you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/995/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>Python SDKs -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/995/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>Before to build -&gt; Before you build</p><p> </p><p>you will -&gt; you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/995/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>Below you will -&gt; Next, you do the following:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/995/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>you will -&gt; you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/995/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>we -&gt; you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/995/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='88'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>you will -&gt; you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/995/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='89'/>

Comment (andrewferlitsch) [inline notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>combine cleanup cells to a single cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/995/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='122'/>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>done</p>

Comment (inardini) [reply notebooks/community/model_registry/vertex_ai_model_registry_automl_model_versioning.ipynb:1]: <p>done</p>

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**